project/build.scala

```diff
@@ -114,6 +114,7 @@ object RhoBuild extends Build {
         license,
 
         libraryDependencies <++= scalazVersion { sz => Seq(
+          shapeless,
```

Comment:
We needed the newer version of shapeless in order to use `HList.unsafePrepend` in `RuleExecutor.scala`.  Unfortunately that method isn't available in shapeless 2.3.0, its only in 2.3.1 and up.  Also, http4s doesn't have a dependency on shapeless, instead it has a dependency on parboiled that in turn depends on shapeless.

```
[master] rho: whatDependsOn com.chuusai shapeless_2.11 2.3.0
[info] Updating rho-examples...
[info] Resolving jline#jline;2.12.1 ...
[info] Done updating.
[info] com.chuusai:shapeless_2.11:2.3.0 [S]
[info]   +-org.parboiled:parboiled_2.11:2.1.3 [S]
[info]     +-org.http4s:http4s-core_2.11:0.15.3 [S]
[info]       +-org.http4s:http4s-blaze-core_2.11:0.15.3 [S]
[info]       | +-org.http4s:http4s-blaze-server_2.11:0.15.3 [S]
[info]       |   +-org.http4s:rho-examples_2.11:0.16.0-SNAPSHOT [S]
[info]       |   
[info]       +-org.http4s:http4s-dsl_2.11:0.15.3 [S]
[info]       | +-org.http4s:rho-examples_2.11:0.16.0-SNAPSHOT [S]
[info]       | 
[info]       +-org.http4s:http4s-jawn_2.11:0.15.3 [S]
[info]       | +-org.http4s:http4s-json4s_2.11:0.15.3 [S]
[info]       |   +-org.http4s:http4s-json4s-jackson_2.11:0.15.3 [S]
[info]       |     +-org.http4s:rho-examples_2.11:0.16.0-SNAPSHOT [S]
[info]       |     
[info]       +-org.http4s:http4s-scala-xml_2.11:0.15.3 [S]
[info]       | +-org.http4s:rho-examples_2.11:0.16.0-SNAPSHOT [S]
[info]       | 
[info]       +-org.http4s:http4s-server_2.11:0.15.3 [S]
[info]         +-org.http4s:http4s-blaze-server_2.11:0.15.3 [S]
[info]           +-org.http4s:rho-examples_2.11:0.16.0-SNAPSHOT [S]
[info]           
```